server/src/main/java/org/elasticsearch/common/settings/AbstractScopedSettings.java

```diff
@@ -729,9 +729,9 @@ private boolean updateSettings(Settings toApply, Settings.Builder target, Settin
                 changed |= toApply.get(key).equals(target.get(key)) == false;
             } else {
                 if (isFinalSetting(key)) {
-                    throw new IllegalArgumentException("final " + type + " setting [" + key + "], not updateable");
+                    throw new IllegalArgumentException("final " + type + " setting [" + key + "], not updatable");
                 } else {
-                    throw new IllegalArgumentException(type + " setting [" + key + "], not dynamically updateable");
+                    throw new IllegalArgumentException(type + " setting [" + key + "], not dynamically updatable");
```

Comment:
Strictly, both are correct, but our docs seem to prefer `updatable`, so this is probably a good change.